web/viewer.css

```diff
@@ -878,16 +923,20 @@ html[dir="ltr"] .dropdownToolbarButton > select {
 html[dir="rtl"] .dropdownToolbarButton > select {
   padding-right: 4px;
 }
-.dropdownToolbarButton > select:hover {
+.dropdownToolbarButton > select:hover,
+.dropdownToolbarButton > select:focus {
   background-color: var(--button-hover-color);
+  color: var(--toggled-btn-color);
 }
 
-.dropdownToolbarButton > select:focus {
-  background-color: var(--button-hover-color);
+.dropdownToolbarButton > select:active {
```

Comment:
Given than none of the "regular" buttons use any `select:active` rules, this feels a little jarring then clicking on the zoom-dropdown; do we *really* need to add this rule?

Reply:
I wanted to make it behave like the 'toggle sidebar', 'find button' and the 'tool' menu button but it didn't work well so I'm reverting it.